### PR TITLE
Adding notes to results and some quick clean up

### PR DESCRIPTION
**Pull request by harsinha**

### CODE REVIEW

File: src/api/routes/test.test.ts [hunk 1]
```diff
@@ -261,9 +285,10 @@ function getValidTestCommand(testTypeId: TestTypeId): TestCommand {
   };
 }
 
-function getValidTestResultsCommand(): TestResultsCommand {
+function getValidTestResultsCommand(str?: string): TestResultsCommand {
```
Comment (kstam) [inline src/api/routes/test.test.ts:288]: maybe better argument naming here?

File: src/infrastructure/persistence/PsqlTestRepository.test.ts [hunk 2]
```diff
@@ -54,17 +54,19 @@ describe('PsqlTestRepository', () => {
     expect(persistedTests[1]).toEqual(test2);
   });
 
-  it('updates a test with results', async () => {
+  it('updates a test with results and notes', async () => {
```
Comment (kstam) [inline src/infrastructure/persistence/PsqlTestRepository.test.ts:57]:
```suggestion
  it('updates a test with results containing notes', async () => {
```

File: src/api/routes/test.test.ts (hunk 1 above)
Comment (harsinha) [reply src/api/routes/test.test.ts:288]: ah good catch. that was done for testing some debugging but forgot to revert. will fix.

Comment (harsinha) [reply src/api/routes/test.test.ts:288]: ah good catch. that was done for testing some debugging but forgot to revert. will fix.

**Pull request merged**